Library/Homebrew/utils/inreplace.rb

```diff
@@ -22,7 +22,10 @@ module Inreplace
     def inreplace(paths, before = nil, after = nil, audit_result = true)
       errors = {}
 
-      Array(paths).each do |path|
+      paths = Array(paths)
+      errors["`paths` (first) parameter"] = ["`paths` was empty"] if paths.blank?
```

Comment:
Thoughts about immediately `raise`ing here?

Reply:
I tend to make a single exit point from the function if it doesn't affect readability and performance. I think for this particular case it is a question of project code style; I don't mind to add a `raise` here

Reply:
Cool, I'm fine leaving it as-is, just wanted to ask 👍